upload/library/SV/WarningImprovements/XenForo/Model/Warning.php

```diff
@@ -928,13 +1035,31 @@ protected function _userWarningPointsIncreased(
 
     public function triggerWarningAction($userId, array $action)
     {
+        if ($action['sv_warning_category_id'] !== 0)
```

Comment:
Caught the first bit this afternoon, will look at _userWarningPointsDecreased() now.